sdk/schemaregistry/azure-schemaregistry-avroserializer/samples/eventhub_receive_integration.py

```diff
@@ -45,12 +45,14 @@ def on_event(partition_context, event):
 
 
 # create a SchemaRegistryAvroSerializer instance
+# TODO: after 'azure-schemaregistry==1.0.0b3' is released, update 'endpoint' to 'fully_qualified_namespace'
```

Comment:
haha, this comment is nice.

(small suggestion that we could put it as a tasks of the issue https://github.com/Azure/azure-sdk-for-python/issues/20628 so that we can't forget it)

Reply:
good idea!

Reply:
I can add it to the issue as well

Reply:
sounds good!